mozilla-release/Makefile.in

```diff
@@ -199,13 +199,10 @@ binaries::
 	@$(MAKE) install-manifests install_manifests=dist/include
 endif
 
-# Cliqz Start
```

Comment:
We still need the `Cliqz Start` comment, as we are downloading the Cliqz Artifacts.